src/sagemaker_containers/_modules.py

```diff
@@ -174,6 +217,7 @@ def run_module(uri, args, env_vars=None, name=DEFAULT_MODULE_NAME, cache=None, w
     _files.download_and_extract(uri, name, _env.code_dir)
 
     install(_env.code_dir)
+    prepare(_env.code_dir, name)
```

Comment:
should this go before `install`?

Reply:
+1 before this was removed, prepare happened before install